components/library/zeromq/zeromq5/zeromq.p5m

```diff
@@ -0,0 +1,30 @@
+#
```

Comment:
Just to be sure: in this context you refer to all (3 or 4) branches of
zeromq that are under review at the moment, or just the 3.* one that was
available before I began about a month ago? Should the
backwards-compatibility ensure that versioned /usr/lib/libzmq*.so* of the
updated zeromq 3.* are still available at those old paths? (During PR these
ABI-versioned links might be removed from the common locations)

--
Typos courtesy of GMail on my Samsung Android
07.04.2016 8:46 пользователь "Alexander Pyhalov" <notifications@github.com>
написал:

> In components/library/zeromq/zeromq5/zeromq.p5m
> <https://github.com/OpenIndiana/oi-userland/pull/1809#discussion_r58825373>
> :
>
> > @@ -0,0 +1,30 @@
> > +#
>
> I'd follow what we've done for gcc versions. And for gcc we just preserved
> old p5m files for runtime. But I would include dependencies on old zeromq
> versions to be respectful to packages dependent on older library/c++/zeromq.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenIndiana/oi-userland/pull/1809/files/3a1854247e2de293fe9526c043c004e8a9744dd1#r58825373>
>